### ISSUE TITLE
Please stabilise the caption text

### ISSUE THREAD

**Issue by XilinJia** [TeamNewPipe/NewPipe#6172]
### Checklist

- [x] I am using the latest version - 0.21.1 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
1. Go to 'https://www.youtube.com/watch?v=Y_ejCmX008s', or basically any video
2. turn on captions
3. choose "auto-generated"

### Actual behaviour

With the auto-generated caption, the text are centrally justified.  This causes the text to shift back and forth when changing lines, making them hard to track by the eye.  

### Expected behavior

The start position of each line of text should stay the same, so perhaps left-justification is ideal, as YouTube does.

### Device info

The behavior appears on various devices, all with Android 9 as I tried.

**Comment by hafizmuhmmadismail** [comment]
Same problem here

**Comment by litetex** [comment]
Hmm this is interesting:
It seems like the subtitles itself determine where they are (→ [ExoPlayer doc](https://exoplayer.dev/doc/reference/com/google/android/exoplayer2/text/Cue.html#textAlignment)):
e.g. https://www.youtube.com/watch?v=BiDvLshi9CY
* English -> Text is centered (and jumps)
![grafik](https://user-images.githubusercontent.com/40789489/116815237-ef982a80-ab5c-11eb-9bce-f6a95a39f2fb.png)
* English/auto-generated -> Text is aligned left 
![grafik](https://user-images.githubusercontent.com/40789489/116815276-0d658f80-ab5d-11eb-98f3-7cd02434c5bd.png)

The above works for YT and the YT app, however the auto-generated text is also centered in the NewPipe app.

Maybe something is lost while parsing the data from YT?

**Comment by snouup** [comment]
Another problem related to subtitle placement when viewing video in landscape mode - on some phones like google pixels, long subtitles can overlap with the front camera, thus a part of the subtitle becomes unreadable.
I advise newpipe to pad the subtitle region by the size of the notification bar to avoid this problem.

**Comment by Angelk90** [comment]
@LeoK987, @hafizmuhmmadismail, @litetex, @snouup:

There is also a font size problem.

Youtube:
![image](https://user-images.githubusercontent.com/20476002/129325205-53c73ba7-74c8-472e-be97-cf6525dbf007.png)

NewPipe:
![image](https://user-images.githubusercontent.com/20476002/129325230-81d024bf-6e71-4440-b9d6-5aceae168844.png)

**Comment by SameenAhnaf** [comment]
@Angelk90 Yes, Go to 'Settings'>'Appearance'>'Captions'>'Caption size and style'.

Also, please avoid pinging so many people from next time.

**Comment by Angelk90** [comment]
@SameenAhnaf : I think there is a problem with the videoplayer subtitles, I put as "very small" font size nothing changed, I closed and reopened the app.

Furthermore there is also the following problem:

![image](https://user-images.githubusercontent.com/20476002/129331847-46fae617-48f1-48c5-bba8-db910e01cb4a.png)

**Comment by Angelk90** [comment]
@TobiGr : There is also a font size problem.

Youtube:
![image](https://user-images.githubusercontent.com/20476002/129325205-53c73ba7-74c8-472e-be97-cf6525dbf007.png)

NewPipe:
![image](https://user-images.githubusercontent.com/20476002/129325230-81d024bf-6e71-4440-b9d6-5aceae168844.png)

I also did this: `'Settings'> 'Appearance'> 'Captions'> 'Caption size and style'.`

But the font does not change, it is very small.
Tested on Tablet.

**Comment by MilaMurry** [comment]
Yeah when is this going to be fixed? It's already 2024 and the issue still persists. Changing the settings in `Settings > Appearance > Captions > Caption size and style`  has no effects and it is not a solution because it will affect other apps. Just adding a slider for the font should have solved the problem.

**Comment by ShareASmile** [comment]
Issue of caption sizes not being changed emphasized in above comments have been addressed with #11806 as for as I am aware of that too in latest v0.27.6, you should check.
